config/application.rb

```diff
@@ -36,7 +36,9 @@ class Application < Rails::Application
     config.api_only = true
 
     # Allow requests from any host
-    config.hosts << ENV["ALLOWED_HOST"]
+    ENV.fetch("ALLOWED_HOSTS") { "" }.split("&").each do |host_regex|
```

Comment:
wow

Reply:
Yeah, corresponds to this uh [thing](https://github.com/go-between/musicbox-terraform/blob/master/staging/app/ecs.tf#L15).  Only way I found to allow both the ip address of the load balancer (which it uses when executing health checks), and also its hostname (which it uses when we make api requests).